sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_component_content_v1/metadata.yaml

```
@@ -0,0 +1,30 @@
+friendly_name: Newtab Component-level Content Table
+description: |-
+    Contains information about content interactions, both organic and sponsored,
+    from the newtab ping.  Each record represents a specific content position during a specific visit.
+
+    Note that this is pulling data from the newtab ping and not the newtab_content ping
+owners:
+  - jsnyder@mozilla.com
+  - gkatre@mozilla.com
```

```suggestion
  - jsnyder@mozilla.com
```
Looks like you may have removed the `owner1` tag completely. You may want to add that and set this with your email. Perhaps the same applies for the view